assets/dev/scss/admin/_promotion.scss

```diff
@@ -0,0 +1,108 @@
+@mixin margin( $top: inherit, $right: inherit, $bottom: inherit, $left: inherit ) {
+	margin-block-start: $top;
+	margin-inline-end: $right;
+	margin-block-end: $bottom;
+	margin-inline-start: $left;
+}
+
+.e-feature-promotion {
+	--e-a-top-bar-height: 50px;
+	--e-a-content-area-spacing: 110px;
+
+	width: 1220px;
+	height: calc( 100vh - var(--e-a-top-bar-height) - var(--e-a-content-area-spacing) );
+	display: grid;
+	grid-template-columns: repeat(2, auto);
+	grid-template-rows: auto;
+	justify-content: space-between;
+	align-items: center;
+	align-content: center;
+	margin: auto;
+	transform: translate( -10px, 40px );
+	text-align: center;
+	font-family: var(--e-a-font-family);
+
+	&_data {
+		grid-column: 1 / 2;
+		grid-row: 1 / 2;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+		align-items: self-start;
+		margin-inline-start: 15px;
+		padding-inline-end: 40px;
+		max-width: 608px;
```

Comment:
What if the screen is smaller?
I am not sure if should be responsive for mobile.

Was it approved by Product?

Reply:
The product asked that this will be mobile friendly
On mobile the design breaks in to column view
The text is above the video